packages/zosfiles/__tests__/__system__/api/methods/download/Download.system.test.ts

```diff
@@ -281,6 +320,45 @@ describe("Download Data Set", () => {
                 expect(fileContents).toEqual(data);
             });
 
+            it("should download a data set and create folders and file in original letter case", async () => {
+                let error;
+                let response: IZosFilesResponse;
+
+                // TODO - convert to UPLOAD APIs when available
+                // upload data to the newly created data set
+                const data: string = "abcdefghijklmnopqrstuvwxyz";
+                const endpoint: string = ZosFilesConstants.RESOURCE + ZosFilesConstants.RES_DS_FILES + "/" + dsname + "(member)";
+                const rc = await ZosmfRestClient.putExpectString(REAL_SESSION, endpoint, [], data);
```

Comment:
Also, we should try our best to implement some of the TODO items, or at least not add more technical debt.
If we feel it's Ok to use the rest client instead of the upload APIs, it's fine. we just need to remove the TODO in that case.